cinema/views.py

```diff
@@ -134,7 +145,11 @@ def get_queryset(self):
 
         return queryset
 
-    def get_serializer_class(self):
+    def get_serializer_class(self) -> Type[
+        MovieSessionListSerializer
+        | MovieSessionDetailSerializer
+        | MovieSessionSerializer
+    ]:
```

Comment:
```suggestion
    def get_serializer_class(self) -> Type[MovieSessionSerializer]:
```

It will be enough because MovieSessionListSerializer and MovieSessionDetailSerializer are children of MovieSessionSerializer